common/faceting.js

```diff
@@ -588,8 +621,102 @@
                         }
                     };
 
+                    // range is defined by the x values of the bar graph because layout.xaxis.type is `linear` or 'category'
+                    function setHistogramRange() {
+                        if (isColumnOfType("timestamp")) {
+                            var minMoment = moment(scope.rangeOptions.absMin.date + scope.rangeOptions.absMin.time, 'YYYY-MM-DDHH:mm:ss');
```

Comment:
The same way that you defined the formats in a constant object, we can do the same in chaise.